src/python/WMCore/WorkQueue/DataStructs/WorkQueueElement.py

```diff
@@ -71,8 +71,12 @@ def __init__(self, **kwargs):
         self.setdefault('OpenForNewData', False)
         # When was the last time we found new data (not the same as when new data was split), e.g. An open block was found
         self.setdefault('TimestampFoundNewData', 0)
-        # Should we check the location of the inputs, or trust the initial values?
+        # Trust initial input and pileup location or not
         self.setdefault('NoLocationUpdate', False)
```

Comment:
I think it might be safer to keep it for this deployment and remove it all together next deployment although I think in most cases, it will be fine. (except something is assigned before the GQ updated. but anyway for agent side it will be safer to include it.  (So I would keep it now and remove it next deployment all together)